eng/pipelines/generate-all-docs.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     variables:
       skipComponentGovernanceDetection: true
     
-    timeoutInMinutes: 120
+    timeoutInMinutes: 1200
```

Comment:
Why is this timeout so large? Do we really need such a large number?

Reply:
@msyyc is trying to test if increasing the timeout can resolve the timeout issue when generating Network SDK docs.